etc/profile-m-z/quodlibet.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for quodlibet
+# Description: Music player and music library manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quodlibet.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${MUSIC}
+
+include allow-bin-sh.inc
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/quodlibet
+mkdir ${HOME}/.config/quodlibet
+mkdir ${HOME}/.quodlibet
+
+whitelist ${MUSIC}
```

Comment:
```suggestion
whitelist ${MUSIC}
whitelist ${DOWNLOADS}
```

Reply:
I liked the idea of allowing only the MUSIC folder, but I'll make the changes.

Reply:
I remarked it because exfalso had it, and without it we break something that was working w/o significant security win. Alternative we could add/keep `whitelist ${DOWNLOADS}` only in exfalso.